ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -135,22 +135,24 @@ StackLayout {
             anchors.bottom: inputArea.top
             anchors.left: inputArea.left
             filter: chatInput.textInput.text
+            cursorPosition: chatInput.textInput.cursorPosition
```

Comment:
I don't think you need this alias

Reply:
Can you elaborate? The aliases here are used to keep the values in sync as they are changing

Reply:
I might be wrong, but it looks like you never access this alias in particular. You don't do `suggestionsBox.cursorPosition` or `this.cursorPosition`. Each time you access `cursorPosition`, you seem to do so directly from the source, which is `filterItem`.

Reply:
> you seem to do so directly from the source, which is filterItem.

`filterItem.cursorPosition` is still updated through the alias from the outside world. Removing the alias removes the "two way databinding", breaking the feature

Maybe I'm missing something.. or if you know a way to make this work without aliases, I'm all ears :)

Reply:
Ok, sorry, I had misunderstood how your alias worked. I tested the branch for myself and you're right, we can't straight up remove the aliases, but I found that aliases are actually overkill for what you want, and honestly make things harder to understand. Using a  normal `property int` is simpler and easier to understand. 

I tried it and it works. I made a branch with it if you want to test: https://github.com/status-im/nim-status-client/compare/feat/better-autocomplete...feat/better-autocompletion-jo?expand=1

Reply:
Thanks @jrainville ! Went with your changes and updated the PR